pkgs/development/python-modules/zipstream-ng/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "zipstream-ng";
+  version = "1.3.4";
+
```

No support for [older Python releases](https://github.com/pR0Ps/zipstream-ng/blob/master/setup.py#L39). 

```suggestion

  disabled = pythonOlder "3.7";
```

My suggestion is to set it to 3.7 as everything else is EOL.